gen-flow.sh

```diff
@@ -1,4 +1,4 @@
 sed '1,2d' src/term-spec.js > dist/term-spec.js.flow;
-echo "declare module 'sweet-spec' {\n" | cat - dist/term-spec.js.flow > out.js;
-echo "\n}" >> out.js;
+echo "// @flow\n" | cat - dist/term-spec.js.flow > out.js;
+echo "\n" >> out.js;
```

Comment:
Got it. We have this issue in core. macro-context.js and browser-sweet.js aren't being checked. eslint-plugin-flowtype started yelling at me about it.